plugins/woocommerce-admin/client/tasks/fills/experimental-products/index.tsx

```diff
@@ -58,12 +60,31 @@ export const Products = () => {
 		experimentLayout,
 	] = useProductTaskExperiment();
 
+	const { isStoreInUS } = useSelect( ( select ) => {
+		const { getSettings } = select( SETTINGS_STORE_NAME );
+		const { general: settings = {} } = getSettings< {
+			general?: { [ key: string ]: unknown };
+		} >( 'general' );
+
+		const country =
+			typeof settings.woocommerce_default_country === 'string'
+				? settings.woocommerce_default_country
+				: 'US';
```

Comment:
I think we should default to non-US experience since the opposite can be slightly risky. The reason being is `subscription` type products are only supported for US countries afaik, so if a non-US store owner sets it up, it may end up not working and confuses the owner.

The other way around isn't too critical, subscription template will go missing which is only sub-optimal, but not breaking.

Reply:
Good point!!! Changed in c5514772483f81508f7ca2bb309c3cb1ccd9caf4